### ISSUE TITLE
Feature: Auto-Miter to apply also on the round part/bend

### ISSUE THREAD

**Issue by lninio** [shaise/FreeCAD_SheetMetal#357]
Implement automatic miters on bends if two bends with a relatively large radius are joined and there is not enough space at the junction for elongation. A typical example is the metal contacts of a 9V battery (6f22). In the image below, I have given the appearance of the contacts of this type of battery, as well as a possible view of the result of processing the coupling of bends in FreeCAD.

![изображение](https://github.com/user-attachments/assets/33547051-b052-47c1-b6bb-ef3ecf23e948)
![20240712_224445](https://github.com/user-attachments/assets/df117ec7-a8a4-4544-8bf9-c29f1e3a71b9)

**Comment by shaise** [comment]
Yes. Basically you want the existing Auto-Miter to apply also on the round part.

**Comment by lninio** [comment]
Yes, it is. Otherwise, I get some redundant gaps around round areas. There are no necessary parameter values for the correct model.
![изображение](https://github.com/user-attachments/assets/14093cf3-ceb4-417d-88db-2e069ba27f3e)

**Comment by lninio** [comment]
Another possible way is to use an "Extend face" and six separate bends for each "petal". But the result will still be wrong, because there is an intersection of bends.

![изображение](https://github.com/user-attachments/assets/7e770b19-2d22-4e07-b066-672581c59520)

**Comment by pierreporte** [comment]
Would be useful for hems (see #302).
